inf-clojure.el

```diff
@@ -978,6 +1006,8 @@ of forms."
   (condition-case nil
       (with-temp-buffer
         (progn
+          ;; TODO: Should it be adjusted for `clojure-ts-mode'
```

Comment:
Yeah, probably. I guess we can infer the major mode to prefer from what's currently active in the open clojure buffers (e.g. we take the first Clojure buffer and check it's mode or something along those lines). I had suggested to add an option for a preferred mode mostly to avoid the need for this inference and to easily pick the right functions from both modes, but there are different ways to approach this.

Reply:
Later in this function `(clojure-forward-logical-sexp)` is used, which might not work as expected in `clojure-ts-mode`, so for now probably it would be better to keep `clojurec-mode` here. BTW, is there a specific reason why `clojurec-mode` is used and not just `clojure-mode`?

Reply:
I don't remember, but if I had to guess - probably `clojure-mode` didn't behave properly in `cljc` files. At any rate - you can just go with `clojure-mode` and `clojure-ts-mode` and we'll know soon enough if this causes any problems.